data/kramdown-rfc2629.erb

```diff
@@ -47,17 +47,29 @@
 <%  if venue -%>
 <%    venue = KramdownRFC::ParameterSet.new(venue) -%>
     <note title="Discussion Venues" removeInRFC="true">
-<%   if mail = venue[:mail] -%>
+<%   mail = venue[:mail] -%>
+<%   homepage = venue[:homepage] -%>
```

Comment:
Maybe just "home"?

Reply:
I'll be guided by you.  I don't see "home" as being clear enough.